Documentation/project-docs/profiling-api-status.md

```diff
@@ -43,4 +43,26 @@ The notes below will help you determine what profiling APIs are safe to use. The
 ##APIs definitely known not to work yet
 ###Windows
 ###Linux
+
+* ICorProfilerInfo:
+ * `SetEnterLeaveFunctionHooks`
+
+* ICorProfilerInfo2:
+ * `SetEnterLeaveFunctionHooks2`
+
+* SetEventMask Flags that currently result in undefined behavior :
+ * `COR_PRF_DISABLE_ALL_NGEN_IMAGES`
```

Comment:
It is likely that https://github.com/dotnet/coreclr/pull/5715 will make `COR_PRF_DISABLE_ALL_NGEN_IMAGES` work well.